### DISCUSSION TITLE
Homeseer Device Donations

### DISCUSSION THREAD

**Discussion by nemith** [zwave-js/zwave-js#3405] [General]
I have some switches that I have or will be replacing and I am wondering what to do with them (stuff in a box, sell on ebay) and wondering if they would be any good to help test/support zwave-js?

They aren't anything special just some homeseer switches.

* HS-WD200+
* HS-WD100+
* HS-WS100+
* HS-WS200+
* HS-FC200+

I'd be willing to pay (reasonable) shipping from the US.

**Comment by AlCalzone** [comment]
@blhoward2 maybe something for you to test? I can't use them since I'm located in EU.

**Reply by blhoward2** [reply]
I’m just seeing this. Apparently I need to come into the Discussions section more often. @nemith If you still have them, I wouldn’t mind one of each to add to our testing pile for when issues come up.

**Reply by nemith** [reply]
I never saw this response until searching something else in my email.  I still have them if you are interested.

**Reply by AlCalzone** [reply]
@nemith if you're willing to ship to Germany, I'll take them. We can talk details via e-mail.
